Mackbellemore/soen-390-team07
# feat(client): Order page UI

#

## Changes this PR introduces

- Table display of orders
- Creating an order modal form



## How to test this PR

- Enter information into the order form
- See the display in the table



## ClickUp reference

CU-n31rda



## Notes for reviewers

- yeet



## Checklist
